### ISSUE TITLE
RIP

### ISSUE THREAD

**Issue by jmarsico** [admsyn/OFPlugin#51]
I think about this plugin almost every day. It was very good to me. Miss u.

**Comment by neelkadia-zz** [comment]
@jmarsico Have you encountered any alternative for this plugins which supports XCode Version 9+ ?

**Comment by jmarsico** [comment]
no, i wish. i've just gotten pretty good at using the project generator (close xcode, rerun PG and include new addon, reopen xcode)

**Comment by neelkadia-zz** [comment]
@jmarsico Same, I'm following same practice.
